### DISCUSSION TITLE
Is there a possibility to switch a second leddevice on and of with json http?

### DISCUSSION THREAD

**Discussion by pjmpessers** [awawa-dev/HyperHDR#567] [Q&A] [answered]
I have a TV-ledstrip that i can switch of and on with json http (LEDDEVICE)
But i also have made a second leddevice connected to my philips hue lights. But i only want to use them with hyperhdr when watching a movie. So i want to turn them on and of as well with the use of json http. Does someone know if this is possible and how?

**Comment by awawa-dev** [comment, marked as answer]
Select wanted instance using API then turn its LEDs off. Two API commands (can be combined into one GET request, everything is available in the 'JSON API' tab).

**Reply by pjmpessers** [reply]
Thank you.
That worked perfectly.
Folowing up a second question. Can you tell me how i can switch on and off the HDR for te hue (Instance 1)?

**Reply by awawa-dev** [reply]
No, HDR processing is tied to a source like USB grabber or flatbuffers (global device, not per instance).

**Reply by pjmpessers** [reply]
Aaah. Thanks. And thanks for the great app hyperHDR.
